tests/system_test.py

```diff
@@ -282,17 +282,22 @@ def __init__(self, args, name=None, expect=EXIT_OK, **kwargs):
         @param expect: Raise error if process status not as expected at end of test:
             L{RUNNING} - expect still running.
             L{EXIT_OK} - expect process to have terminated with 0 exit status.
-            L{EXIT_FAIL} - expect process to have terminated with exit status 1.
+            L{EXIT_FAIL} - expect process to have terminated with exit status >= 1.
             integer    - expected return code
         @keyword stdout: Defaults to the file name+".out"
         @keyword stderr: Defaults to be the same as stdout
         """
         self.name = name or os.path.basename(args[0])
         self.args = args
+        if expect is None:
```

Comment:
A `RuntimeError` is raised if except is `None`

Reply:
Getting rid of None is a good idea - the expect parameter has defined an explicit set of valid values that does not include None.

Personally I would consider any test passing None for expect a coding error, and rather than raising an exception I'd just:

assert expect is not None, "Process argument expect cannot be None"